tests/monotouch-test/monotouch-test.csproj

```diff
@@ -761,7 +762,6 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Security\openssl_crt.der">
-        <LogicalName>monotouchtest.Security.openssl_crt.der</LogicalName>
```

Comment:
@spouliot I would love to but I can't repro anymore. Might have an updated VS that fixed it.